lib/cisco_node_utils/radius_server.rb

```diff
@@ -73,7 +132,13 @@ def ==(other)
     end
 
     def auth_port
-      config_get('radius_server', 'auth-port', @name)
+      if platform == :ios_xr
+        @auth_port
+      else
+        val = config_get('radius_server', 'auth-port', ip: @name)
+        val = val[0] if val.is_a?(Array)
+        val
```

Comment:
since auth-port is not `multiple` on Nexus, the Array check shouldn't be needed:

```ruby
def auth_port
  platform == :ios_xr ? @auth_port : config_get('radius_server', 'auth-port', ip: @name)
end
```